### DISCUSSION TITLE
Different fonts for different languages

### DISCUSSION THREAD

**Discussion by dodio12138** [squidfunk/mkdocs-material#6552] [Q&A]
I would like to ask if there is a configuration option to set the corresponding fonts for different languages.

**Comment by kamilkrzyskow** [comment]
Different languages? I guess you mean coding languages. I assume you're using the `pymdownx.highlight` highlighter for code:
https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#highlight
Looking at the options, there is `pygments_lang_class`:
https://facelessuser.github.io/pymdown-extensions/extensions/highlight/#options

It adds a `language-codeIdentifier` class to the parent wrapper where the `css_class` option is also used. The language prefix comes from the `language_prefix` option.

So after enabling `pygments_lang_class` you should be able to target it in CSS and change fonts based on the language used.
As for using custom fonts, there are examples you can reference here: 
https://github.com/mkdocs-material/examples/tree/master/examples (source)
https://github.com/mkdocs-material/examples/tree/gh-pages (zipped)
`font-builtin.zip` and `font-custom.zip`.

If you meant normal languages and not code languages then please provide the i18n solution you're using.

**Reply by dodio12138** [reply]
Thank you very much for your reply, I understand how to configure it.
